auditor/src/client/mod.rs

```diff
@@ -452,13 +601,27 @@ impl AuditorClientBlocking {
         Ok(self
             .client
             .get(format!(
-                "{}/record?state=stopped&since={}",
+                "{}/record?stop_time[gte]={}",
                 &self.address, encoded_since
             ))
             .send()?
             .error_for_status()?
             .json()?)
     }
+
+    /// Get custom records using filters - start_time, stop_time, runtime_filters
```

Comment:
Documentation: We should probably specify a little bit more in detail how the `query_params` string works. But I think here is not the right place. But we could link to a general endpoint documentation. Let me come back to you once I figure out where we want to place this. In addition, we should also link to the QueryBuilder, as this should be the preferred way to build up queries.